muhasin/index.js

```diff
@@ -0,0 +1,20 @@
+let snackServings = 4
+let guests = 5
+console.log(snackServings + guests);
+console.log(snackServings - guests);
+console.log(snackServings * guests);
+console.log(snackServings / guests);
+
+
+let drinks = 10
+let resultOne = (guests * snackServings) / drinks
+
+
+let monkeys = 15
+let banana = 9
+console.log(monkeys / banana);
+
+let monkeyfights = 20
```

Comment:
The solution of 7. is wrong!!! 
you have to make a variable and then shod the result using console.log()
the solution is 
const monkeyfights = 20;
const resultTwo = (monkeys + bananas) * monkeyfights;
console.log(resultTwo);